src/mlpack/core/cv/k_fold_cv.hpp

```diff
@@ -56,8 +56,8 @@ template<typename MLAlgorithm,
          typename PredictionsType =
              typename MetaInfoExtractor<MLAlgorithm, MatType>::PredictionsType,
          typename WeightsType =
-             typename MetaInfoExtractor<MLAlgorithm, MatType,
-                 PredictionsType>::WeightsType>
+             typename MetaInfoExtractor<MLAlgorithm, MatType, PredictionsType>::
```

Comment:
I think the previous one is easier to read, not sure if we can avoid a break once a namespace or something like that is used.

Reply:
I'll have to customize the sentence braking, I'll see what can be done.